tutorial/practicals/LFRic/building_code/4_psydata/solutions/value_range_check_all_transform.py

```diff
@@ -65,8 +63,8 @@ def trans(psy):
         schedule = invoke.schedule
 
         # Apply the transformation
-        nan_check.apply(schedule, {"region_name": ("time_evolution",
-                                                   str(invoke_name))})
+        value_range_check.apply(schedule, {"region_name": ("time_evolution",
+                                                           str(invoke_name))})
 
         # Just as feedback: show the modified schedule, which should have
         # a new node at the top:
```

Comment:
I don't think we need the final `return` any more.

Reply:
Removed.